manuscript/glossary.md

```diff
@@ -0,0 +1,80 @@
+  an international open data infrastructure to publish and disseminate
+  biodiversity information.
+
+* **iDigBio** (Integrated Digitized Biocollections, https://www.idigbio.org) is a
+  project funded by the National Science Foundation that facilitates the
+  digitization of natural history collections, and provides data and images for
+  biological specimens.
+
+#### File Formats
+
+* **CSV** (Comma-Separated Values) and **TSV** (Tab-Separated Values) are plain text
+  file formats used to store tabular data where each row is represented by a line
+  in the file, and each field (column) is separated by a comma for CSV or by the
+  Tab character for TSV.
+
+* **FASTA** is a simple file format used to represent sequences of nucleotides or
+  amino-acids in plain text making it easy to manipulate programmatically.
+
+* **HDF5** (Hierarchical Data Format) is an open-source binary file format designed
+  to store large amount of data (and their associated metadata) by providing a
+  hierarchical structure that could be compared to how a hard drive is organized
+  with directories and files.
+
+* **NetCDF** (Network Common Data Form) is an open-source binary file format
+  designed to store large datasets in array-oriented scientific data as
+  typically used in geosciences.
+
+#### Programming and Algorithms
+
+* Web **APIs** (Application Programming Interface) provide ways to query
+  programmatically databases through the internet. They notably allow users to
+  retrieve and work with a small slice of a large dataset.
+
+* **MapReduce** is a style of programming designed to work with large datasets in
+  parallel computing environments. Such programs are composed of a Map procedure
+  where the dataset is sliced into several pieces, and a Reduce procedure where
+  summary operations are applied to each of the slices.
+
+* **MD5** is a Message Digest algorithm used in cryptographic hash functions often
+  used to verify data integrity. A cryptographic hash function converts a
+  "message" (e.g., passwords, file content) into an encrypted
+  value. Cryptographic hash functions are easy to compute from the message, but
+  it should be impossible to recover the message from the output, and any
+  modifications to the message should also modify the output.
+
+* **SQL** (Structure Query Language) is a programming language used to interact with
+  relational database management systems.
+
+#### Hardware
+
+* **mega-, giga-, tera-, peta-bytes** are units of digital information and are used
+  to measure the size of datasets or the storage media. Originally a byte was
+  the minimum amount of memory needed to store a single character of text in a
+  computer. The prefixes mega-, giga-, tera- and peta- refer to the
+  international system of units for the multiple of the unit, and correspond to
+  $10^{6}$, $10^{9}$, $10^{12}$, and $10^{15}$ abbreviated M, G, T and P
+  respectively.
+
```

Comment:
this might be a good place for some examples ... or perhaps an illustrated line on the log scale, from b --> Eb could include images along the lines of a character, an image, RAM, hard drive, the amount of publicly available curated data in the world, the amount public uncurated data, the amount of data that is about to be lost in notebooks, the amount of data hidden by obscurity, and the amount of data required to understand the meaning of life

something like this, but in a figure

![image](https://cloud.githubusercontent.com/assets/464871/10182792/15781044-66ee-11e5-886b-466879809b51.png)


from http://cs.sru.edu/~mullins/cpsc100book/module02_introduction/module02-05_introduction.html